paasta_tools/cli/cmds/status.py

```diff
@@ -416,7 +416,7 @@ def report_status_for_cluster(
                 for deployed_instance in deployed_instances
                 if (deployed_instance in http_only_instances or use_api_endpoint)
             ]
-            if any(return_code != 200 for return_code in return_codes):
+            if any(return_codes):
```

Comment:
wait, why not filter out 200's?

Reply:
Let me double check, I think the function can't ever return the 200 at the moment